test/helpers/kubectl.go

```diff
@@ -1195,6 +1195,28 @@ func (kub *Kubectl) Logs(namespace string, pod string) *CmdRes {
 		fmt.Sprintf("%s -n %s logs %s", KubectlCmd, namespace, pod))
 }
 
+// LogsWithPrevious returns a CmdRes with containing the resulting metadata from the
```

Comment:
```suggestion
// LogsWithPrevious returns a CmdRes with metadata from the
```
Same for the sibling functions below.